src/Ninject.Test/Integration/ReadOnlyKernelTests.cs

```diff
@@ -194,6 +226,22 @@ public void TryGet_ServiceAndNameAndParameters_ReturnsNullIfMultipleBindingsExis
                 bindings.Should().HaveCount(1);
             }
 
+            [Fact]
+            public void TryGet_ServiceAndNameAndParameters_ReturnsNullIfMultipleBindingsExistForResolvedService()
```

Comment:
The two tests for the non-generic `TryGet` overloads _failed_ before this change. They both returned the service resolved from the _first_ matching binding.